build.sbt

```diff
@@ -199,3 +199,5 @@ publishArtifact := false
 
 publishMavenStyle := true
 publishTo in ThisBuild := Some(Resolver.file("gh-pages-repo", baseDirectory.value / "repo"))
+
+libraryDependencies += "com.lihaoyi" %% "upickle" % "0.5.1"
```

Comment:
This should be removed. We have dropped `uPickle` in favor of Argonaut, which supports all platforms we want to use.